l10n_es_intrastat_report/readme/CONTRIBUTORS.rst

```diff
@@ -12,3 +12,7 @@
 
   * Harald Panten
   * Valentin Vinagre
+
+* `GreenIce <https://www.greenice.com>`__:
+
+  * Fernando La Chica <fernandolachica@gmail.com>
```

Comment:
If you put the company, it seems inconsistent to put a personal email inside. And I recommend you to not put for avoiding "spam"